angular.json

```diff
@@ -93,5 +97,8 @@
       "projectType": "application"
     }
   },
-  "defaultProject": "kahla"
-}
+  "defaultProject": "kahla",
+  "cli": {
+    "analytics": "b4eba425-1a0f-4d1e-959a-f7c5d15a63ca"
```

Comment:
What is that?

Reply:
It seems like a annoymous information analytics from angular and can be removed safely.
https://angular.io/cli/analytics